### ISSUE TITLE
Remove simple/general distinction

### ISSUE THREAD

**Issue by wpetry** [padrinoDB/ipmr#69]
I think this decision could be hidden from the user, since it appears to be implemented only for computational efficiency.

Under the hood, the simple approach could be used after checking user inputs that:
- all kernels have only continuous states
- only one continuous state variable is used

**Comment by levisc8** [comment]
I agree that it could be done. I don't think I'll have time to implement such a feature though. 

As fairly simple check within `make_ipm()` is probably the best way to achieve this (I think).
